README.md

```diff
@@ -1,111 +1,129 @@
-[![Documentation](https://readthedocs.org/projects/restic/badge/?version=latest)](https://restic.readthedocs.io/en/latest/?badge=latest)
+<p align="center">
+  <img src="https://restic.readthedocs.io/en/latest/_static/logo.png" alt="Restic logo">
+</p>
+
+<p align="center">
+  <strong>Restic</strong> – <strong>Fast</strong>, <strong>efficient</strong>, and <strong>secure</strong> backup program
+  <br>
+  <a href="https://restic.net/">Website</a> |
+  <a href="https://restic.readthedocs.io/">Documentation</a> |
+  <a href="https://forum.restic.net/">Forum</a> |
+  <a href="https://github.com/restic/restic/issues">Issues</a>
+</p>
```

Comment:
I like the idea of heaving a nice header. IMHO
* the logo should be smaller and 
* the link to issues is not necessary as the github link to issues is just above the logo.

Reply:
I will reduce the size of the logo. I think the Issues link should remain, as it is useful when the repository is forked or mirrored.